style/index.html

```diff
@@ -476,10 +476,28 @@
               </div>
             </div>
             <div class='inputs'>
-              <input id="fileupload" type="file" name="file" data-url="/file/upload">
-              <div class='filenames'></div>
-              <input class='description-field' placeholder='File Nickname'>
-              <div class='button upload'>Upload</div>
+              <div class='step1 step'>
+                <div class='step-text'>Select a file</div>
+                <div class='capsule'>
+                  <div class='pod'>
+                    <div class='file-upload-container'>
+                    <span class='file-upload-label button'><i class='fa fa-hand-o-up'></i>Select File</span>
+                    <input id="fileupload" type="file" name="file" data-url="/file/upload">
+                    </div>
+                  </div>
+                  <div class='pod'>
+                    <div class='file-upload-filename'>No File Selected</div>
+                  </div>
+                </div>
+              </div>
+              <div class='step2 step'>
+                <div class='step-text'>Give it a name</div>
+                <input class='description-field' placeholder='File Nickname'>
+              </div>
+              <div class='step3 step'>
+                <div class='step-text'>Upload to Sia</div>
+                <div class='button upload'>Upload</div>
+              </div>
```

Comment:
It will be nice once the generated html is gone. I'm also increasingly in support of breaking the frontend into its own repo, though we'll need a good way to keep the projects integrated.